test/spec/spectest.cpp

```diff
@@ -46,34 +36,37 @@ using namespace WasmEdge;
 
 // Preprocessing for set up aliasing.
 void resolveRegister(std::map<std::string, std::string> &Alias,
-                     rapidjson::Value &CmdArray,
-                     rapidjson::Document::AllocatorType &Allocator) {
-  rapidjson::Value::ValueIterator ItMod = CmdArray.Begin();
-  for (rapidjson::Value::ValueIterator It = CmdArray.Begin();
-       It != CmdArray.End(); ++It) {
-    const auto CmdType = It->GetObject()["type"].Get<std::string>();
+                     simdjson::dom::array CmdArray) {
```

Comment:
Why do we need to remove the reference of `CmdArray`?

Reply:
Didn't mean to remove it, will add it back

Reply:
I have added it back in https://github.com/WasmEdge/WasmEdge/pull/2659/commits/b903dff5d269911bd3a841257eba8f95fe440630